dialect.html

```diff
@@ -49,16 +89,20 @@ <h3 class="Box-title f5 text-bold text-gray-dark">Start Dialect Integration Test
                             <input id="dialect-test-name" type="text" name="test" value="" placeholder="test/unit/ht/test_ht_dialect_sql.rb" class="form-control input-monospace" >
                         </dt>
                         <dt>
-                            <dt><label for="name-text">Test Regex</label></dt>
-                            <input id="dialect-test-regex" type="text" name="filter" value="" placeholder="/works for a trivial materialization/" class="form-control input-monospace" >
```

Comment:
these / / dont need to be typed out, they'll get auto wrapped so dont need to make it seem like the user needs to input em

Reply:
I just wanted to make it clear that it's a regex